toolchain/check/handle_export.cpp

```diff
@@ -62,7 +62,11 @@ auto HandleParseNode(Context& context, Parse::ExportDeclId node_id) -> bool {
     context.emitter()
         .Build(node_id, ExportRedundant)
         // Use the location of the export itself, not the exported instruction.
-        .Note(context.insts().GetLocId(inst_id), ExportPrevious)
+        //
+        // TODO: This is a bit too magical. It prevents GetAbsoluteNodeIdImpl()
```

Comment:
```suggestion
        // TODO: Look at making this more explicit. It prevents GetAbsoluteNodeIdImpl()
```

I don't feel good calling other people's code "a bit too magical", maybe we can phrase more objectively?

Reply:
Done, I think, PTAL